src/components/representative-form/index.tsx

```diff
@@ -67,6 +89,20 @@ const RepresentativeForm = ({
     }
   }, [values, hasEURepresentative]);
 
+  const showLegalNotice = () => (
```

Comment:
Seems unnecessary to create a separate function just to render a component.

Reply:
we have `renderForm()` tho

Reply:
I saw that. It's unfortunate, since it makes no sense to separate it either.